### DISCUSSION TITLE
New .js.org domain?

### DISCUSSION THREAD

**Discussion by abhay7dev** [ReplAPI-it/ReplAPI.it-NodeJS#51] [Improvements/R&D] [answered]
Just a suggestion.
At the moment, the docs for ReplAPI.it it is https://replapi-it-docs.vercel.app/. However, you can in fact get a subdomain of `.js.org` for free from https://js.org/ (github repo [here](https://github.com/js-org/js.org)). It is much shorter than `vercel.app` and may look better having a link like `replapi-it.js.org` rather than the current url.

**Comment by rayhanadev** [comment, marked as answer]
@EpicGamer007 I LOVE your thinking :3!

However, do you know if I can connect my site from Vercel to a js.org subdomain? I use Vercel mainly because of their Edge Network and easy deployment optimized for Next.js, but quickly scanning through the homepage it seems js.org wants people to use Github Pages.

**Reply by abhay7dev** [reply]
I will take a look, maybe search their issues and pull requests for anything related to vercel

**Reply by abhay7dev** [reply]
@RayhanADev , i have taken a look at the domains and they seem to have a few vercel domains like this one: https://github.com/js-org/js.org/pull/5738 this pull has someone submitting a .vercel.app domain so you 100% will be able to.

**Reply by rayhanadev** [reply]
Sweeet! I'll take a look today :)

**Reply by rayhanadev** [reply]
@EpicGamer007 update: I opened a pr on the repo and I'm going through the process :)

**Reply by abhay7dev** [reply]
nice

**Reply by rayhanadev** [reply]
@EpicGamer007 done 🎉. Check out the new domain at [https://replapi-it.js.org](https://replapi-it.js.org/).

**Reply by abhay7dev** [reply]
noice
